cache/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Sorry I forgot to delete. I normally use cache folder for twig templates, like this:
```php
        // Set up Twig Environment with cache folder
        $environment = new Twig_Environment($loader, array(
            'cache' => $cache,
        ));
```
where $cache is path to Cache dir.